--- a/setup.py
+++ b/setup.py
@@ -1382,22 +1382,6 @@ class PyBuildExt(build_ext):
         else:
             missing.append('gdbm')

-        # Unix-only modules
-        if host_platform not in ['win32']:
-            # Steen Lumholt's termios module
-            exts.append( Extension('termios', ['termios.c']) )
-            # Jeremy Hylton's rlimit interface
-            if host_platform not in ['atheos']:
-                exts.append( Extension('resource', ['resource.c']) )
-            else:
-                missing.append('resource')
-
-            nis = self._detect_nis(inc_dirs, lib_dirs)
-            if nis is not None:
-                exts.append(nis)
-            else:
-                missing.append('nis')
-
         # Curses support, requiring the System V version of curses, often
         # provided by the ncurses library.
         panel_library = 'panel'
